File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-04/plagiarized/L5/01/L5.ts
package IR;

class L5 {
    static print(mil: number): void {
        for (let i = 0; i < 10; i++) {
            console.log(`${mil}\t\t${mil * 1.609}`);
            mil++;
        }
    }

    static main(args: string[]): void {
        let mil = 1;
        console.log("Miles\t\tKilometers");
        console.log("-------------------------------");
        this.print(mil);
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-04/plagiarized/L5/01/L5.kt
package IR

fun print(mil: Int) {
    for (i in 0 until 10) {
        println("$mil\t\t${mil * 1.609}")
        mil++
    }
}

fun main(args: Array<String>) {
    var mil = 1
    println("Miles\t\tKilometers")
    println("-------------------------------")
    print(mil)
}

